src/app/page.tsx

```diff
@@ -8,17 +8,30 @@ import Link from 'next/link';
 export default function Home() {
     return (
         <Container>
-            <div className="d-flex mt-3">
-                {data.categories.map((category) => (
-                    <Link
-                        className="category-labels"
-                        key={category}
-                        href={buildCategoryUrl(category)}
-                    >
-                        {category}
-                    </Link>
-                ))}
-            </div>
+            <Navbar expand="lg" className="main-navbar mobile-navbar">
+                <Container>
+                    <Navbar.Toggle aria-controls="main-navbar-nav" />
+                    <Navbar.Offcanvas id="main-navbar-nav" placement="end">
+                        <Offcanvas.Header closeButton>
+                            <Offcanvas.Title className="d-flex align-items-center"></Offcanvas.Title>
+                        </Offcanvas.Header>
+                        <Offcanvas.Body>
+                            <Nav className="justify-content-end flex-grow-1 pe-3">
+                                {data.categories.map((category) => (
+                                    <Link
+                                        className="category-labels"
+                                        key={category}
+                                        href={buildCategoryUrl(category)}
+                                    >
+                                        <Nav.Link href="/">{category}</Nav.Link>
```

Comment:
Why is there two links? Come on man! Pay attention!

Reply:
I was in a doubt of choosing among the two and left both. removing the link now.